pokedex/data/csv/location_area_prose.csv

```diff
@@ -9,7 +9,7 @@ location_area_id,local_language_id,name
 8,9,
 9,9,
 10,9,
-11,9,Route 207
+11,9,"Route 207"
```

Comment:
These quotes shouldn't be necessary. Could you make sure this round-trips through `pokedex load` / `pokedex dump`?

Reply:
Ah, I wish I had known about the `pokedex dump` command earlier! I'll use that to get rid of the quotes when I fix the still-missing location areas (not sure how that happened 😬)

Reply:
Oh no! :( Yeah, uhhh, i guess we should probably document it a little better.